### DISCUSSION TITLE
Ваши идеи и предложения по проекту

### DISCUSSION THREAD

**Discussion by denekben** [denekben/WorldDomination#2] [Ideas]
init      =^_^=

**Comment by betr0dalf** [comment]
It would be nice to add the functionality of adding players as friends so that you can play with them later

**Reply by denekben** [reply]
Yeah, I don't think it will be difficult

**Comment by NikolayMagarin** [comment]
A game mode where you play against AI-powered bots.
**AI! AI! More AI!**
